styles/main.css

```diff
@@ -309,13 +309,38 @@ td:nth-of-type(7) {
 .burger{
     display:block;
 }
-header nav ul {
-    height:0;
-    overflow:hidden;
-    height:auto;
-}
-.burger~#burger-toggle {
-    left:5px;
-    top:10px;
-    transform:rotate(45deg);
+header nav input { display: none; }
+
+@media screen and ( max-width: 768px ) {
+    header nav ul {
+        height:0;
+        overflow:hidden;
+        padding-top: 0;
+        padding-bottom: 0;
+        margin-top: 1em;
+        margin-bottom: 1em;
+
+        transition: height 500ms ease-in-out;
+    }
+
+    header nav #burger-toggle:checked ~ ul {
+        height: 110px;  
+    }

```

Comment:
When the burger-toggle checkbox is checked, make the menu (its subsequent sibling) have a proper height